ballerina/channel_csv_io.bal

```diff
@@ -135,3 +161,40 @@ isolated function getWritableCSVChannel(WritableChannel writableChannel) returns
     }
     return writableCSVChannel;
 }
+
+isolated function readHeadersFromCsvFile(string path) returns string[]|Error {
+    stream<string[], Error?>|Error csvContent = fileReadCsvAsStream(path);
+        if csvContent is Error {
```

Comment:
formatting issue?

Reply:
Yes. was updated in here [4b07dfa](https://github.com/ballerina-platform/module-ballerina-io/pull/827/commits/4b07dfaf3558ebc118cc2615b36e48d3565b24e9)